transports/webrtc/Cargo.toml

```diff
@@ -1,29 +1,28 @@
 [package]
-name = "libp2p-webrtc"
-version = "0.6.0-alpha"
 authors = ["Parity Technologies <admin@parity.io>"]
+categories = ["asynchronous", "network-programming"]
 description = "WebRTC transport for libp2p"
-repository = "https://github.com/libp2p/rust-libp2p"
-license = "MIT"
 edition = "2021"
+keywords = ["libp2p", "networking", "peer-to-peer"]
+license = "MIT"
+name = "libp2p-webrtc"
+repository = "https://github.com/libp2p/rust-libp2p"
 rust-version = { workspace = true }
-keywords = ["peer-to-peer", "libp2p", "networking"]
-categories = ["network-programming", "asynchronous"]
+version = "0.6.0-alpha"
```

Comment:
Happy for these to be alphabetic. But please in a separate pull request to keep the noise in this one low.

Reply:
Cross-referencing tracking issue: https://github.com/libp2p/rust-libp2p/issues/4043